sale_exception/views/sale_view.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" ?>
+<!-- License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+        <record id="action_sale_test_tree" model="ir.actions.act_window">
```

Comment:
It would be great to tidy up formatting

Reply:
done in a separate commit to backport it easily if needed